hack/addons/kustomize/cilium/kustomization.yaml.tmpl

```diff
@@ -29,5 +29,6 @@ helmCharts:
       enabled: false
       relay:
         enabled: false
+        priorityClassName: system-cluster-critical
```

Comment:
This is only applicable to the CRS generation and will have no effect either for that because the line above disables relay in CRS!

You want to change the Helm addon values in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/charts/cluster-api-runtime-extensions-nutanix/addons/cni/cilium/values-template.yaml.

Reply:
Thanks.

Reply:
Fixed.